src/runtime/plugin.client.ts

```diff
@@ -35,24 +35,34 @@ function createFetchClient(
     retry: false,
 
     async onRequest(context) {
-      let csrfToken = readCsrfCookie(authentication.csrfCookie)
-
-      if (!csrfToken.value) {
-        await $fetch(authentication.csrfEndpoint, {
-          baseURL: authentication.baseUrl,
-          credentials: 'include',
-          retry: false,
-        })
-
-        csrfToken = readCsrfCookie(authentication.csrfCookie)
+      // todo: move this to interceptors
```

Comment:
interceptors are not really needed here unless we have more layers to apply, so I would suggest either removing todo comments or creating a new issue instead